core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitOperationAttribute.java

```diff
@@ -0,0 +1,59 @@
+package com.scalar.db.transaction.consensuscommit;
+
+import com.scalar.db.api.Put;
+import java.util.Map;
+import java.util.Optional;
+
+/** A class to manage the operations attributes for Consensus Commit. */
+public final class ConsensusCommitOperationAttribute {
```

Comment:
Added `ConsensusCommitOperationAttribute` to manage the operations attributes for Consensus Commit.